cgroupfs-mount

```diff
@@ -41,6 +41,10 @@ for sys in $(awk '!/^#/ { if ($4 == 1) print $1 }' /proc/cgroups); do
 	fi
 done
 
+if [ -e /sys/fs/cgroup/memory/memory.use_hierarchy ]; then
+	echo 1 > /sys/fs/cgroup/memory/memory.use_hierarchy
+fi
```

Comment:
Aha, thanks for digging up that systemd link -- I meant to go find that but hadn't had the time yet.

I'd like to see two minor changes to this, if you don't mind.  I'd like to move it down below the example `/proc/cgroups` comment below, and I'd like to add some comments to point to why we're doing it, something like:

```bash
# example /proc/cgroups:
# ...

# enable cgroups memory hierarchy, like systemd does (and lxc desires)
# https://bugs.debian.org/940713
# https://github.com/systemd/systemd/blob/d910f4c2b2542544d7b187a09605da7a0f220837/src/core/cgroup.c#L3011
if [ -e ... ]; then
	...
fi

exit 0
```

Reply:
I changed the `systemd` ref to the latest release to help track down any future changes they may decide on.